apiserver/facades/client/subnets/subnets_test.go

```diff
@@ -114,7 +174,7 @@ func (s *SubnetsSuite) TestNewAPIWithBacking(c *gc.C) {
 	agentAuthorizer := s.authorizer
 	agentAuthorizer.Tag = names.NewMachineTag("42")
 	facade, err = subnets.NewAPIWithBacking(
-		apiservertesting.BackingInstance,
+		&stubBacking{apiservertesting.BackingInstance},
```

Comment:
this is done in preparation that we don't need to update the apiservertesting stubs, but can update them here and slowly move away from the stubs.